README.md

```diff
@@ -79,8 +79,8 @@ Requires docker:
 ```sh-session
 make test-standalone
 
-# Change the test timeout (useful on slower systems):
-TEST_TIMEOUT=10m make test-standalone
+# Disable running tests in parallel and change the test timeout (useful on slower systems):
+DISABLE_PARALLEL_TESTS=true TEST_TIMEOUT=10m make test-standalone
```

Comment:
Rather than DISABLE_PARALLEL_TEST, can we expose setting the parallelism? Then, if you want to disable it, you can set it to 1?